SelfReflection1-EmilyRowen.qmd

```diff
@@ -0,0 +1,61 @@
+---
+title: "Emily Rowen Self Reflection 1"
+format: html
+editor: visual
+---
+
+**About you**
+
+1.  Please tell me a little bit about your research interests and how you imagine integrating spatial data in your research.
+
+    I've been thinking about how energy-ag and water-ag conflicts are occurring in Idaho and how the community capitals framework might inform how these conflicts are more multidimensional than current approaches portray.
+
+    One way that I would like to integrate spatial data is in energy-ag conflicts there are the sites of energy developments (wind/solar) but then there isn't clear discussion on which communities around these developments are included in "stakeholder engagement" processes. In the water-ag conflict of the Eastern Snake River Plain I think it would be interesting to map out where junior/senior water rights holders are located and see if this is influencing the conflict at all.
+
+2.  How do you learn best, and what do you feel is your biggest strength as a student?
+
+    In R classes I've taken before, I love some written out directions that allow me to move at my own pace (so homeworks might be the most beneficial to me?). I'm definitely a visual learner. I also like to collaborate with other students in class on mini-challenges where the teacher shows us an example, but then we have to figure out the next example ourselves (I like talking through these with peers more than independently figuring it out).
+
+3.  If you want to, you can set a goal for this semester for something you want to work on in the context of this course (outside of the topic matter itself) – for example, using StackOverflow to get answers to analysis questions or using git to track progress on homeworks. If you choose to set this type of goal, what can you do to help support yourself in working toward that goal? How can I help you to meet this goal?
+
+    I want to understand how the GitHub pipeline works and feel confident using it to share my code with others.
```

Comment:
Hopefully, the pipeline will become more clear through our homework process! It is fairly straightforward to share your finished code from a published product -- you can just upload all your files to GitHub and not worry about pushing/pulling/committing like we will with the homework.